.github/workflows/plugins-test.yaml

```diff
@@ -118,6 +118,8 @@ jobs:
         run: bash test/plugin/run.sh netty-socketio-scenario
       - name: Run rabbitmq-scenario (12)
         run: bash test/plugin/run.sh rabbitmq-scenario
+      - name Run gson-scenario (6)
```

Comment:
Your format is still not right. You need to pay more attention to details :)

Reply:
i am sorry. i've checked.